enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/UnionAcceptanceTest.scala

```diff
@@ -39,7 +39,7 @@ class UnionAcceptanceTest extends ExecutionEngineFunSuite with CypherComparisonS
       |M.a as B
     """.stripMargin
 
-    val result = succeedWith(Configs.Interpreted - Configs.EnterpriseInterpreted, query)
+    val result = succeedWith(Configs.CommunityInterpreted, query)
```

Comment:
I know you didn't add `CommunityInterpreted` in this commit, but while you're at it...
I think it's a bit weird to use the word "Community" here, it's not a runtime.
Maybe rename `Interpreted` with `AllInterpreted` or `SlottedAndInterpreted` and `CommunityInterpreted` to `Interpreted`?